make-xges.py

```diff
@@ -390,9 +436,9 @@ def save(self):
 
 def main(argv):
     parser = argparse.ArgumentParser(description='convert a BigBlueButton presentation into a GES project')
-    parser.add_argument('--start', metavar='SECONDS', type=float, default=0,
-                        help='Seconds to skip from the start of the recording')
-    parser.add_argument('--end', metavar='SECONDS', type=float, default=None,
+    parser.add_argument('--start', metavar='TIME', type=str, default='0',
+                        help='Start point in the recording (seconds, or mm:ss, hh:mm:ss, dd:hh:mm:ss)')
+    parser.add_argument('--end', metavar='TIME', type=str, default=None,
```

Comment:
Could you get the same effect here by changing `type=str` to `type=my_time_format`?

Looking at the documentation (https://docs.python.org/3.9/library/argparse.html#type), this should work and avoid the need for the other changes to the code.

Reply:
My friend, I tried. I need more help for that. I lost 40 minutes and then let it go.